diesel/src/row.rs

```diff
@@ -11,7 +11,7 @@ use deserialize::{self, FromSql};
 /// [`FromSqlRow`]: ../deserialize/trait.FromSqlRow.html
 pub trait Row<DB: Backend> {
     /// Returns the value of the next column in the row.
-    fn take(&mut self) -> Option<&DB::RawValue>;
+    fn take<'a>(&'a mut self) -> Option<RawValue<'a, DB>>;
```

Comment:
That lifetime should not be required
```suggestion
    fn take(&mut self) -> Option<RawValue<DB>>;
```